src/main/java/com/trillia/entities/BusinessObject.java

```diff
@@ -0,0 +1,119 @@
+package com.trillia.entities;
+
+import org.hibernate.search.annotations.*;
+
+import javax.persistence.*;
+import javax.xml.bind.annotation.XmlElement;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Created by mmathew on 26/05/16.
+ */
+
+@Entity
+@Table(name = "trl_business_objects",uniqueConstraints = {@UniqueConstraint(columnNames = {"tenantId","objectCode"})})
+@Indexed(index = "BusinessObject")
```

Comment:
Please mention the @MultiTenant annotation for all entities , so that the repository layer functions properly. For this table, we should have -- @MultiTenant(level = MultiTenant.Levels.ONE_N_GLOBAL) , so that the GLOBAL data is visible to all tenants.